src/Features/LanguageServer/ProtocolUnitTests/Completion/CompletionResolveTests.cs

```diff
@@ -28,13 +28,19 @@ void M()
             using var workspace = CreateTestWorkspace(markup, out var locations);
             var tags = new string[] { "Class", "Internal" };
             var completionParams = CreateCompletionParams(locations["caret"].Single(), "\0", LSP.CompletionTriggerKind.Invoked);
+            var commitCharacters = new string[]
+                {
+                    " ", "{", "}", "[", "]", "(", ")", ".", ",", ":",
+                    ";", "+", "-", "*", "/", "%", "&", "|", "^", "!",
+                    "~", "=", "<", ">", "?", "@", "#", "'", "\"", "\\"
```

Comment:
we're hardcoding this list?

Reply:
updated to pick the default list
(this is from the reverted PR)